app/gui/src/model/execution_context/synchronized.rs

```diff
@@ -308,6 +313,16 @@ impl model::execution_context::API for ExecutionContext {
     fn rename_method_pointers(&self, old_project_name: String, new_project_name: String) {
         self.model.rename_method_pointers(old_project_name, new_project_name);
     }
+
+    fn set_mode(&self, mode: ExecutionEnvironment) -> BoxFuture<FallibleResult> {
```

Comment:
sometimes the var with exec env is called `environment`, sometimes `mode`. This is a minor inconsistency.

Reply:
I hope I spotted the last ones I missed. Will double-check again.